integration/insert_compat_test.go

```diff
@@ -18,17 +18,20 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/AlekSi/pointer"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
 
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
 type insertCompatTestCase struct {
-	insert     bson.D                   // required
+	insert     []any                    // required, slice of bson.D to be insert
+	ordered    *bool                    // defaults to true
```

Comment:
FWIW, I think we should not use `*bool` that has three values if we use only two. The simplest solution could be `ordered bool // defaults to false` and `options.InsertMany().SetOrdered(tc.ordered)`

Reply:
Just in case this change was made after this [comment](https://github.com/FerretDB/FerretDB/pull/1673/files/aaf4b39f782bf7bb09a4a3bfe9ffb1eff31a900f#r1058492595). But yeah if we prefer simpler approach then let's do this!

Reply:
fixed